Source/Core/Core/ConfigLoaders/BaseConfigLoader.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright 2016 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#include "Common/CommonPaths.h"
```

Comment:
`<cstring>`, `<list>`, `<map>`, `<string>`, and `"Common/CommonTypes.h"` need to be included.